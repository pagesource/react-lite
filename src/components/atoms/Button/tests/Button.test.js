import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Button from '../index';

configure({ adapter: new Adapter() });

describe('<Button />', () => {
  let ButtonComponent = '';
  beforeEach(() => {
    ButtonComponent = shallow(<Button>Test</Button>);
  });

  test('should render correctly', () => {
    expect(ButtonComponent).toMatchSnapshot();
  });
});
