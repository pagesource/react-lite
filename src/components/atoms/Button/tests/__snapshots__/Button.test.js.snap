// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> should render correctly 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  ",
          "
  background: transparent;
  border-radius: 3px;
  border: 2px solid palevioletred;
  color: palevioletred;
  margin: 0.5em 1em;
  padding: 0.25em 1em;

  ",
          [Function],
          "
",
          ";
",
        ],
      },
      "displayName": "Styled(Button)",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": [Function],
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  Test
</StyledComponent>
`;
